src/App/PropertyLinks.cpp

```diff
@@ -1793,6 +1859,88 @@ void PropertyLinkSub::getLinks(std::vector<App::DocumentObject *> &objs,
     }
 }
 
+static void _getLinksTo(const App::PropertyLinkBase* prop,

```

Comment:
This should be in an anonymous namespace. Eight parameters is also straining the limits of reasonableness, and one of them is a trailing boolean for even more warning flags. Finally, it's got a cognitive complexity of 36, which is quite high. I know it's far from the worst thing we've merged as part of this project, but that's an awful lot of red flags. I'm open to suggestions for cleanup.

Reply:
Hmm.  Called three times; from PropertyLinkSub, PropertyXLink, PropertyXLinkSubList.  Only one loop, but a bunch of conditionals.  Could squash the parms into a struct, but we still need them so I'm not sure it really helps.  Can fix the bool easily enough.  Desk Check time ...

Reply:
First take on this now in #15501

Reply:
Sounds great, I can live with that.